source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Infrastructure/Persistence/Repositories/MarketParticipantRepository.cs

```diff
@@ -88,7 +88,9 @@ public Task<List<MarketParticipant>> GetActiveEnergySuppliersAsync()
             return _chargesDatabaseContext
                 .MarketParticipants
                 .Where(mp => mp.BusinessProcessRole == MarketParticipantRole.EnergySupplier)
-                .Where(m => m.IsActive)
+                .Where(m =>
+                    m.Status == MarketParticipantStatus.Active ||
+                    m.Status == MarketParticipantStatus.Inactive)
```

Comment:
Is this intended or should it be Passive?

```suggestion
                    m.Status == MarketParticipantStatus.Passive)
```